### ISSUE TITLE
Cygwin: "Unsupported Arch. Force compilation errors"

### ISSUE THREAD

**Issue by m-ghaoui** [karthick18/inception#3]
Running on:

CYGWIN_NT-6.1-WOW64 1.7.7(0.230/5/3) i686 Cygwin

$ make
gcc -g -Wall -m32  -c -o inception.o inception.c
In file included from inception.c:109:
inception.h:218:2: #error "Unsupported Arch. Force compilation errors"
inception.c: In function `enter_limbo':
inception.c:548: error:`inception_thoughts' undeclared (first use in this function)
inception.c:548: error: (Each undeclared identifier is reported only once
inception.c:548: error: for each function it appears in.)
inception.c: In function `shared_dream_level_1':
inception.c:1779: warning: implicit declaration of function`nop_fill'
inception.c:1780: error: `fischers_thoughts' undeclared (first use in this function)
make: **\* [inception.o] Error 1

**Comment by karthick18** [comment]
Yeah this is expected as its only supposed to work on Linux (x86/64, mips, arm) and Apple. Not on Windows :).
The reason is the instruction op-code buffer that does the inception and uses int 0x80 or syscall to enter the kernel. I don't think that would work in Windows. So you are free to port it to Windows to make the inception work :)
